# SINGLE-CELL GENOMIC ANALYSIS USING MACHINE LEARNING

## BACKGROUND

**Single-cell genomics** is a field of biology focused on analyzing the genomes of **individual cells**. This approach allows scientists to study the unique genetic characteristics and variations within a population of cells, revealing insights that are often masked when studying cells in bulk. By examining individual cells, researchers can better understand cellular **diversity**, **identify new cell types**, and investigate how **genetic differences** contribute to various biological processes and diseases. 

**RNA-seq (RNA sequencing)** is a next-generation sequencing (NGS) technique that sequences and quantifies the entire RNA content known as the transcriptome in a biological sample. The process involves isolating RNA from cells or tissues, converting it into complementary DNA (cDNA) using the enzyme reverse transcriptase, and then sequencing the cDNA fragments to determine both the identity and abundance of RNA molecules present.

**Machine Learning (ML)** is area which is focused on **creating systems** and **algorithms** that can automatically **learn** from and make **predictions** or **decisions** based on data, without being explicitly programmed for each specific task.

**Ewing Sarcoma** is a rare malignant (cancerous) tumor that typically affects children, adolescents and young adults, most commonly between 10 and 20 years old. It usually develops in the long bones (such as the femur, tibia, or humerus), as well as the pelvis, ribs, and spine. The tumor likely originates from primitive nerve cells or stem cells in bone or soft tissue.
It is the second most common bone cancer in young people, after osteosarcoma.

## Objectives 

### Main:
Assess and compare the capability of conventional and ML-based pipelines in scRNAseq data analysis.

### Specific:

1. Evaluate clustering ability and accuracy of each pipeline using quantitative metrics and low-dimensional visualizations.
2. Determine the efficiency of each method in identifying biologically relevant and statistically significant Ewing Sarcoma genetic markers.
3. Assess and compare the reproducibility of clustering, marker gene identification, and trajectory inference across three scRNA-seq datasets using both methods.

## RESULTS
---

## REFERENCES

1. **Durer, S., Gasalberti, D. P., & Shaikh, H.** (2024). *Ewing Sarcoma*. In **StatPearls \[Internet]**. Treasure Island (FL): StatPearls Publishing.
   [PubMed: 32644609](https://pubmed.ncbi.nlm.nih.gov/32644609/)

2. **Subbiah, V., Somaiah, N., & Trent, J. C.** (2019). *Ewing sarcoma: Current management and future approaches*. **Journal of Clinical Oncology**, 37(27), 2900–2909.
   [https://doi.org/10.1200/JCO.19.00978](https://doi.org/10.1200/JCO.19.00978)

3. **Gaspar, N., Hawkins, D. S., Dirksen, U., Lewis, I. J., Ferrari, S., Le Deley, M. C., ... & Ladenstein, R.** (2015). *Ewing sarcoma: Current management and future approaches—Results of the EICESS and Euro-E.W\.I.N.G. clinical trial groups*. **Pediatric Blood & Cancer**, 62(8), 1248–1258.
   [https://doi.org/10.1002/pbc.25552](https://doi.org/10.1002/pbc.25552)

4. **Crompton, B. D., Stewart, C., Taylor-Weiner, A., Alexe, G., Kurek, K. C., Calicchio, M. L., ... & Meyerson, M.** (2014). *The genomic landscape of pediatric Ewing sarcoma*. **Nature Genetics**, 46(5), 503–509.
   [https://doi.org/10.1038/ng.2938](https://doi.org/10.1038/ng.2938)


5. **Haque, A., Engel, J., Teichmann, S. A., & Lönnberg, T.** (2017). *A practical guide to single-cell RNA-sequencing for biomedical research and clinical applications*. **Genome Medicine**, 9(1), 75.
   [PubMed: 28821273](https://pubmed.ncbi.nlm.nih.gov/28821273/)
   [Full Text](https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-017-0467-4)

6. **Zheng, G. X. Y., et al.** (2022). *Single-cell RNA sequencing technologies and applications*. **Clinical and Translational Medicine**, 12, e703.
   [PubMed: 35352511](https://pubmed.ncbi.nlm.nih.gov/35352511/)

7. **Qu, H.-Q., Kao, C., & Hakonarson, H.** (2024). *Single-Cell RNA Sequencing Technology Landscape in 2023*. **Stem Cells**, 42(1), 1–12.
   [PubMed: 37934608](https://pubmed.ncbi.nlm.nih.gov/37934608/)

8. **Luecken, M. D., & Theis, F. J.** (2019). *Current best practices in single-cell RNA-seq analysis: a tutorial*. **Molecular Systems Biology**, 15(6), e8741.
   [PubMed: 37566049](https://pubmed.ncbi.nlm.nih.gov/37566049/)
---

